### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by dublx**
Upgrade to Fastify v3;
Upgrade to GraphQL v15;
Updated other dependencies;
Fixed tests where the expected return is now different;
Added new values to `lab test --globals` to overcome error messages about leaking;

### CODE REVIEW

File: packages/trail-hapi-server/package.json [hunk 1]
```diff
@@ -38,21 +38,21 @@
     "coverage": "lab -l -c test",
     "depcheck": "npx depcheck --ignores=config",
     "start": "node start.js",
-    "test": "lab test --globals __extends,__assign,__rest,__decorate,__param,__metadata,__awaiter,__generator,__exportStar,__values,__read,__spread,__spreadArrays,__await,__asyncGenerator,__asyncDelegator,__asyncValues,__makeTemplateObject,__importStar,__importDefault,__classPrivateFieldGet,__classPrivateFieldSet"
+    "test": "lab test --globals __core-js_shared__,CSS,regeneratorRuntime,__extends,__assign,__rest,__decorate,__param,__metadata,__awaiter,__generator,__exportStar,__values,__read,__spread,__spreadArrays,__await,__asyncGenerator,__asyncDelegator,__asyncValues,__makeTemplateObject,__importStar,__importDefault,__classPrivateFieldGet,__classPrivateFieldSet"
   },
   "dependencies": {
-    "@hapi/hapi": "^19.1.0",
-    "@hapi/inert": "^6.0.1",
+    "@hapi/hapi": "^19.2.0",
```
Comment (mcollina) [inline packages/trail-hapi-server/package.json:44]: Can you also bump Hapi to latest? So we can ship a new major.

Comment (dublx) [reply packages/trail-hapi-server/package.json]: Done in 1ba66a91dd9d3fa8d96b9215c27a4f4243f019ef

File: packages/trail-core/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nearform/trail-core",
-  "version": "4.0.1",
+  "version": "5.0.0",
```
Comment (mcollina) [inline packages/trail-core/package.json:3]: Please bump the major after landing, straight to master. Thanks

**Comment by dublx** [comment]
I think thats all dependecies updated now @mcollina 
Ok to merge?

**Comment by mcollina** [comment]
Please fix the version number in this PR first and delete the tags.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**